src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -56,12 +56,12 @@ const isValidAddress = (protocolVersion: ProtocolVersion, addressStr: string) =>
 const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion) => {
   try {
     if (protocolVersion === ProtocolVersion.ipv4) {
-      return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
+      return RESERVED_LOCAL_HOST_IPS.ipv4.includes(ip) || RESERVED_CATCH_ALL_IPS.ipv4.includes(ip);
     } else {
       const ipv6Address = new Address6(ip);
       return (
-        compareIPV6Addresses(new Address6(RESERVED_LOCAL_HOST_IPS.ipv6), ipv6Address) ||
-        compareIPV6Addresses(new Address6(RESERVED_CATCH_ALL_IPS.ipv6), ipv6Address)
+        findIPV6AddressInReservedAddresses(RESERVED_LOCAL_HOST_IPS.ipv6, ipv6Address) ||
```

Comment:
```suggestion
        hasReservedIpv6Address(ipv6Address) 
```
and the function looks at all the reserved addresses since we don't care which one it is